types/replicatedservice/v2/replicatedservice.py

```diff
@@ -0,0 +1,194 @@
+"""Defines a ReplicatedService type by creating both a Service and an RC.
```

Comment:
It may make sense to have a separate file that is part of the "type" definition, since the changelog isn't really necessary to the schema of the template.

Would be curious to see what @jackgr thinks about this, as he's given the semver stuff a lot of thought and is a strong proponent.